src/index.js

```diff
@@ -0,0 +1,84 @@
+import './style.css';
+// tasks array
+let tasks = [
+];
```

Comment:
- [ ] as this is the first milestone, you need to create an array of objects and populate your page from the array, I can see you have done some requirements that are not part of the milestone, kindly follow the milestone-specified requirement
![Screenshot from 2023-03-28 18-04-30](https://user-images.githubusercontent.com/66097371/228315726-516eca47-b94b-43e5-8d63-ee5bbd741034.png)